CHANGES.txt

```diff
@@ -39,6 +39,11 @@ Breaking Changes
 Changes
 =======
 
+- Introduce support for UNION ALL to produce the combined result of two
+  queries, e.g.:
+
+    select * from t1 UNION ALL select * from t2
```

Comment:
please capitalize keywords:
`SELECT * FROM t1 UNION ALL SELECT * FROM t2`

to make the example more explicit you could also explicitly say:
```
SELECT id FROM t1
UNION ALL
SELECT id FROM t2;
```